docs/MigrationGuide.md

```diff
@@ -104,3 +104,13 @@ to fallback to default description with payment reference:
 
 **Note**: according to [PayPal Plus documentation](https://developer.paypal.com/docs/api/payments/#definition-transaction)
 length of the description must be up to 127 characters, thus length of `reference` must be up to 116 characters.
+
+### To v0.8.0
+
+New service version is capable of adding prefix for productName of paypal `Item`. Prefix is constructed 
+through a product attribute. Attribute can be configured through `prefixProductNameWithAttr` optional 
+configuration property in application.yml file. Product name with prefix should not exceed 127 characters. 
+In case if prefixed productName exceeds the limit, it will discard the product name prefix.
```

Comment:
According to documentation, the Paypal API will automatically truncate the name -> no need to discard

Reply:
But in this case, it will discard the product name. I assume truncate will start from the rear of the string.

Reply:
For now I would not truncate if Paypal does it for us. Less logic less issues.

Reply:
7a392e38709a23661ae076c280a282ede94b319d